src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1148,6 +1149,28 @@ public void cannotCheckOutWhenItemLimitIsReachedForBookMaterialTypeAndLoanTypeIs
     assertThat(secondBookTypeItem, hasItemStatus(AVAILABLE));
   }
 
+  @Test
+  public void canCheckOutWithdrawnItem() {
+    final IndividualResource withdrawnItem = itemsFixture
+      .basedUponSmallAngryPlanet(ItemBuilder::withdrawn);
+
+    assertThat(withdrawnItem.getJson(), hasJsonPath("status.name", "Withdrawn"));
+
+    final IndividualResource response = loansFixture
+      .checkOutByBarcode(withdrawnItem, usersFixture.steve());
+
+    assertThat(response.getJson(), allOf(
+      hasOpenStatus(),
+      hasJsonPath("action", "checkedout"),
+      hasJsonPath("item.status.name", "Checked out"),
+      hasJsonPath("itemId", withdrawnItem.getId().toString()),
+      hasJsonPath("item.id", withdrawnItem.getId().toString())
+    ));
+
+    assertThat(itemsClient.getById(withdrawnItem.getId()).getJson(),
+      hasJsonPath("status.name", "Checked out"));
+  }
```

Comment:
Could this be extracted and re-used? It might make the intent clearer and reduce the need to duplicate the `Checked out` item status.

Reply:
Moved to `ItemMatchers` class.

Reply:
Thank you for extracting these out into matchers